src/libstd/sys/unix/process/magenta.rs

```diff
@@ -188,3 +181,124 @@ extern {
 
     pub fn launchpad_vmo_from_file(filename: *const c_char) -> mx_handle_t;
 }
+
+// Errors
+
+#[allow(unused)] pub const ERR_INTERNAL: mx_status_t = -1;
+
+// ERR_NOT_SUPPORTED: The operation is not implemented, supported,
```

Comment:
This and all the following comments that document these constants should use the triple slash `///` which rustdoc recognizes as a doc comment and uses in the generated HTML.